### DISCUSSION TITLE
resize plugin messes with fixed positioning - can't pin element to edge of viewport

### DISCUSSION THREAD

**Discussion by majorminors** [jspsych/jsPsych#1120] [Q&A]
Hello,

I'm having trouble with some CSS positioning following resizing with the resize plugin. I can't figure out how the resize plugin is working its magic to fix it.

The desired endstate is to pin an element to the edges of the viewport. I have been using `position: fixed` and `bottom: 0`, `right: 0`. This is an alteration of  @becky-gilbert 's audio-response plugin--trying to pin the recording indicator to the edge of the viewport - visible, but discreet and not interfering with the visual stimulus. You can see the css here:

https://github.com/majorminors/vocal-stroop-js/blob/master/jspsych-6.1.0/plugins/jspsych-image-audio-response.js#L231

After resizing with the resize plugin however, the element I had pinned ends up floating in the middle of the screen. It looks like it might be pinned to the edges of a div, but I'm not savvy enough to figure out exactly what's happening here.

I created a test script to demonstrate what I mean. It uses @becky-gilbert 's audio-response plugin to illustrate, since that's what I'm working with. If you clone this repo:
https://github.com/majorminors/vocal-stroop-js
and run `test_issue.html` in a browser.

Without resizing, the recording indicator is pinned to the bottom right of the viewport, as demonstrated in the first two trial blocks of `test_issue.html`. After resizing, the recording indicator appears to have migrated centrally, as in the last two trial blocks of `test_issue.html`.

Any advice would be greatly appreciated. I simply need to get that element pinned out of the way!

Warm regards,

Dorian

**Comment by jodeleeuw** [comment]
I'm converting this to a discussion for now. We'll try to help out as soon as possible but time is tight right now. Sorry!

**Reply by majorminors** [reply]
No worries. Thanks!
